_maps/templates/boarding/syndicate/mako_carrier.dmm

```diff
@@ -603,6 +655,13 @@
 /obj/machinery/computer/ship/viewscreen,
 /turf/open/floor/carpet/red,
 /area/ruin/powered/nsv13/boarding_interior)
+"Eb" = (
+/obj/structure/closet/crate,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
```

Comment:
```suggestion
/obj/effect/spawner/lootdrop/maintenance/six,
```
Ditto regarding the ruin loot, you could have one or two in there but this is probably a bit much (seeing as it's also two crates)